sdk/python/kfp/compiler/pipeline_spec_builder.py

```diff
@@ -155,8 +155,7 @@ def build_task_spec_for_task(
                         _additional_input_name_for_pipeline_channel(input_value)
                     )
                     assert component_input_artifact in parent_component_inputs.artifacts, \
-                        'component_input_artifact: {} not found. All inputs: {}'.format(
-                            component_input_artifact, parent_component_inputs)
+                        f'component_input_artifact: {component_input_artifact} not found. All inputs: {parent_component_inputs}'
```

Comment:
FYI, in google3, there's a `line-too-long` lint error on any line that exceeds 80 characters. So many of these styles was in consideration to prepare a smooth sync between the two in case development happens in google3.
I personally also keep a 80-character width limit, which works for the number of window-splits in my IDE, and I never need to scroll sideway to see the whole line. 

https://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
That being said, I'm okay we drop this convention.